<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoadMeasurements</class>
 <widget class="QDialog" name="LoadMeasurements">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>501</width>
    <height>541</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Load Measurements</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../../../../../../../../QRapBusy/Coverage.png</normaloff>../../../../../../../../QRapBusy/Coverage.png</iconset>
  </property>
  <widget class="QWidget" name="layoutWidgetFF">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>481</width>
     <height>29</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="layoutFileFormat">
    <item>
     <widget class="QLabel" name="lFileFormat">
      <property name="text">
       <string>File Format to import:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="cBoxFileType">
      <item>
       <property name="text">
        <string>GSM Drivetest</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>CW space delimited Lat, Lon and RxLev</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>CW *.csv with Timestamp, Lat, Lon, and RSSI </string>
       </property>
      </item>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="lFiles">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>310</y>
     <width>101</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>File(s) to load: </string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lEditFileToLoad">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>330</y>
     <width>391</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pButtonBrowse">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>330</y>
     <width>81</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Browse...</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pButtonLoad">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>470</y>
     <width>151</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Load Measurements</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidgetDouble">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>481</width>
     <height>95</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayoutsdataLinks">
    <item row="0" column="0">
     <widget class="QLabel" name="lMeasType">
      <property name="text">
       <string>Measurement Type:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QComboBox" name="cBoxMeasType"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="lMeasSource">
      <property name="text">
       <string>Measurement Source:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QComboBox" name="cBoxMeasSource"/>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="lPosSource">
      <property name="text">
       <string>Position Source:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QComboBox" name="cBoxPosSource"/>
    </item>
   </layout>
  </widget>
  <widget class="QListWidget" name="listWidgetFiles">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>370</y>
     <width>321</width>
     <height>161</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pButtonClose">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>500</y>
     <width>151</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Close / Cancel</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>170</y>
     <width>271</width>
     <height>128</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="lTxCell">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="text">
       <string>Transmitting Cell:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QComboBox" name="cBoxCell">
      <property name="enabled">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="lFrequency">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="text">
       <string>Transmitting Frequency:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QDoubleSpinBox" name="dSpinBoxFreq">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="minimum">
       <double>30.000000000000000</double>
      </property>
      <property name="maximum">
       <double>23000.000000000000000</double>
      </property>
      <property name="value">
       <double>900.000000000000000</double>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="lSensitivity">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="text">
       <string>Sensitivity:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QDoubleSpinBox" name="dSpinBoxSensitivity">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="minimum">
       <double>-150.000000000000000</double>
      </property>
      <property name="maximum">
       <double>0.000000000000000</double>
      </property>
      <property name="value">
       <double>-110.000000000000000</double>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="lMobileHeight">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="text">
       <string>    Mobile Height:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QDoubleSpinBox" name="dSpinBoxMobileH">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="minimum">
       <double>0.500000000000000</double>
      </property>
      <property name="maximum">
       <double>30.000000000000000</double>
      </property>
      <property name="value">
       <double>1.300000000000000</double>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
